utils/sendGridEventWebhook.js

```diff
@@ -0,0 +1,120 @@
+const sgMail = require("@sendgrid/mail");
+const { SecretManagerServiceClient } = require("@google-cloud/secret-manager");
+const { EventWebhook, EventWebhookHeader } = require("@sendgrid/eventwebhook");
+const { processEventWebhook } = require("./firestore");
+const { getResponseJSON } = require("./shared");
+
+const _testSendEmail = async (res) => {
+    console.log("Start sending email");
+
+    try {
+        // Add SendGrid apiKey to test
+        sgMail.setApiKey("");
+
+        // Update data to test
+        const msg = {
+            personalizations: [
+                {
+                    to: "",
+                    custom_args: {
+                        connect_id: "cnid1",
+                        token: "tk1",
+                        notification_id: "nid1",
+                        gcloud_project: process.env.GCLOUD_PROJECT,
+                        attempt: "1",
+                    },
+                },
+                // {
+                //     to: "",
+                //     custom_args: {
+                //         connect_id: "cnid2",
+                //         token: "tk2",
+                //         notification_id: "nid2",
+                //         gcloud_project: process.env.GCLOUD_PROJECT,
+                //         attempt: "1",
+                //     },
+                // },
```

Comment:
Is this commented out code needed?

Reply:
It's used for testing send to multiple emails